src/app/contributors/contributors.service.ts

```diff
@@ -0,0 +1,15 @@
+import { HttpClient } from '@angular/common/http';
+import { inject, Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { ContributorsModel } from './contributors.model';
+
+@Injectable({ providedIn: 'root' })
+export class ContributorsService {
```

Comment:
Suggestion: We should set this up as `GitHubService` instead. This will allow us to extend this service to support other features like fetching project stars count, or any other vanity metrics.